src/components/Body/Portafolio/Portafolio.js

```diff
@@ -1,42 +1,32 @@
 import React, { Component } from 'react';
+import Ability from '../../../actions/Ability';
+
+import ItemPortafolio from './ItemPortafolio'
 import front_end from '../../../assets/front_end.png';
 import best_practices from '../../../assets/best-practice.jpg';
 import algorithms from '../../../assets/algorithms_new.jpg';
 
 class Portafolio extends Component {
+
+  constructor(personalAbilities){
```

Comment:
Same as above, I'm gonna suggest a React tutorial to understand how a component should work dude.